Siri/L2/Level2.java

```diff
@@ -0,0 +1,29 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Level2 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        ArrayList<String> arrList = new ArrayList<String>();
+        String line = "    __________________________________________________\n";
+        System.out.println(line + "     Hi there. I'm Siri \n" + "     How may I help you? \n" + line);
+
+        while(true) {
```

Comment:
Perhaps it would be better to put spaces before the parenthesis? in adherence to the java coding standard.